Source build failing at llvm install

Error is 
```
cd scratch/llvm-julia-20.1.2-0/build_Release+Asserts && mkdir -p [buildroot]/usr-staging/llvm-julia-20.1.2-0/build_Release+Asserts[buildroot]/usr/tools/lit && cp -r [buildroot]/deps/srccache/llvm-julia-20.1.2-0/llvm/utils/lit/{*.py,*.toml,lit/} [buildroot]/usr-staging/llvm-julia-20.1.2-0/build_Release+Asserts[buildroot]/usr/tools/lit/ && [buildroot]/deps/scratch/cmake-3.30.1-linux-x86_64/bin/cmake -DCMAKE_INSTALL_PREFIX="[buildroot]/usr-staging/llvm-julia-20.1.2-0/build_Release+Asserts[buildroot]/usr" -P cmake_install.cmake && cp [buildroot]/usr-staging/llvm-julia-20.1.2-0/build_Release+Asserts[buildroot]/usr/bin/lld [buildroot]/usr-staging/llvm-julia-20.1.2-0/build_Release+Asserts[buildroot]/usr/tools
cp: cannot stat '[buildroot]/deps/srccache/llvm-julia-20.1.2-0/llvm/utils/lit/{*.py,*.toml,lit/}': No such file or directory
make[1]: *** [[buildroot]/deps/llvm.mk:331: [buildroot]/usr-staging/llvm-julia-20.1.2-0/build_Release+Asserts.tar] Error 1
```
Run at https://buildkite.com/julialang/julia-master-scheduled/builds/1210/steps/canvas?sid=01981c8c-dec7-464e-b622-e6510f40bac4

Presumably due to the change in https://github.com/JuliaLang/julia/commit/d150fdf0550c2e5c7ea418f6522038b9140ccf5b
```
diff --git a/deps/llvm.mk b/deps/llvm.mk
index ec606a98c8484..7c4acff9e537b 100644
--- a/deps/llvm.mk
+++ b/deps/llvm.mk
@@ -306,8 +312,8 @@ endif
 	echo 1 > $@
 
 LLVM_INSTALL = \
-	cd $1 && mkdir -p $2$$(build_depsbindir) && \
-	cp -r $$(SRCCACHE)/$$(LLVM_SRC_DIR)/llvm/utils/lit $2$$(build_depsbindir)/ && \
+	cd $1 && mkdir -p $2$$(build_depsbindir)/lit && \
+	cp -r $$(SRCCACHE)/$$(LLVM_SRC_DIR)/llvm/utils/lit/{*.py,*.toml,lit/} $2$$(build_depsbindir)/lit/ && \
 	$$(CMAKE) -DCMAKE_INSTALL_PREFIX="$2$$(build_prefix)" -P cmake_install.cmake
 ifeq ($(OS), WINNT)
 LLVM_INSTALL += && cp $2$$(build_shlibdir)/$(LLVM_SHARED_LIB_NAME).dll $2$$(build_depsbindir)
```
Not obvious to me why it's erroring, @Keno?

--- a/deps/llvm.mk
+++ b/deps/llvm.mk
@@ -313,7 +313,9 @@ endif
 
 LLVM_INSTALL = \
 	cd $1 && mkdir -p $2$$(build_depsbindir)/lit && \
-	cp -r $$(SRCCACHE)/$$(LLVM_SRC_DIR)/llvm/utils/lit/{*.py,*.toml,lit/} $2$$(build_depsbindir)/lit/ && \
+	cp $$(SRCCACHE)/$$(LLVM_SRC_DIR)/llvm/utils/lit/*.py $2$$(build_depsbindir)/lit/ && \
+	cp $$(SRCCACHE)/$$(LLVM_SRC_DIR)/llvm/utils/lit/*.toml $2$$(build_depsbindir)/lit/ && \
+	cp -r $$(SRCCACHE)/$$(LLVM_SRC_DIR)/llvm/utils/lit/lit $2$$(build_depsbindir)/lit/ && \
 	$$(CMAKE) -DCMAKE_INSTALL_PREFIX="$2$$(build_prefix)" -P cmake_install.cmake
 ifeq ($(OS), WINNT)
 LLVM_INSTALL += && cp $2$$(build_shlibdir)/$(LLVM_SHARED_LIB_NAME).dll $2$$(build_depsbindir)

